### PR TITLE
Virtual check-in

### PR DESCRIPTION
**Pull request by cabeese**
Update team check-in page to not require a volunteer to scan the QR code of a team. We will remove this functionality for in-person events.

### CODE REVIEW

File: client/components/TeamCheckin/imports/TeamCheckInVirtually.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        self.setState({
+          modalOpen: true,
+          modalHeader: "Failed to check your team :(",
+          modalContent: `${error.reason}. If this continues to be a problem, please contact support.`
+        });
+      } else {
+        self.setState({
+          modalOpen: true,
+          modalHeader: "You are checked in!",
+          modalContent: `You may procede to the game page`
```
Comment (RvanB) [inline client/components/TeamCheckin/imports/TeamCheckInVirtually.jsx]: Typo, should be "proceed"

**Review by RvanB — CHANGES_REQUESTED** [review]
Fix typo above, rebase with master before merging. Worked really well!

File: client/components/TeamCheckin/imports/TeamCheckInVirtually.jsx (hunk 1 above)
Comment (cabeese) [reply client/components/TeamCheckin/imports/TeamCheckInVirtually.jsx]: fixed, ty!

**Comment by cabeese** [comment]
@RvanB thanks for testing and the comments. I pushed up the grammar fix. I also decided to drop the commit that would enforce team leaders to be checked in since we no longer lock down functionality for them. Ready for final review

**Review by RvanB — APPROVED** [review]

**Pull request merged**